snow/engine/snowman/bootstrap/metrics.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 type metrics struct {
-	numFetched, numDropped, numAccepted prometheus.Counter
```

Comment:
`numDropped` was only `Inc()`ed immediately prior to `FATAL`ing - so I don't think this metric is really important to keep.

Reply:
Pulled into a separate PR: #2858